libs/java.json

```diff
@@ -214,6 +220,7 @@
   {"id":"java.maria-db4j","imports":["ch.vorburger.mariadb4j."],"name":"MariaDB4j","repo":"vorburger/MariaDB4j","tags":["sql"],"tech":["storage"],"status":"model-ready"},
   {"id":"java.minio","imports":["io.minio."],"name":"Minio","repo":"minio/minio-java","tags":["s3"],"tech":["storage"],"status":"model-ready"},
   {"id":"java.mongodb","imports":["org.mongodb"],"name":"mongo-java-driver","repo":"mongodb/mongo-java-driver","tags":["mongodb"],"tech":["storage"],"status":"awaiting-model"},
+  {"id":"java.mybatis","imports":["org.mybatis"],"name":"MyBatis SQL Mapper Framework for Java","repo":"mybatis/mybatis-3","tags":["sql"],"tech":["storage"],"status":"awaiting-model"},
```

Comment:
Alright, maybe the name should be just "MyBatis" because there is no way we can display what you have.